### PR TITLE
Update robot framework

### PR DESCRIPTION
**Pull request by henryju**

### CODE REVIEW

File: its/src/test/kotlin/org/sonarlint/intellij/its/BaseUiTest.kt [hunk 1]
```diff
@@ -140,8 +140,8 @@ open class BaseUiTest {
         }
         searchField().text = fileName
         val fileList = jList(JListFixture.byType(), Duration.ofSeconds(5))
-        waitFor(Duration.ofSeconds(5)) { fileList.items.isNotEmpty() }
-        fileList.selectItem(fileList.items[0], false)
+        waitFor(Duration.ofSeconds(5)) { fileList.collectItems().isNotEmpty() }
+        fileList.clickItemAtIndex(0)
         waitFor(Duration.ofSeconds(10)) { editor(fileName).isShowing }
         waitBackgroundTasksFinished()
```
Comment (damien-urruty-sonarsource) [inline its/src/test/kotlin/org/sonarlint/intellij/its/BaseUiTest.kt:146]: A bit of duplication here

**Review by damien-urruty-sonarsource — APPROVED** [review]
Minor comments, else :+1:

File: its/src/main/resources/META-INF/plugin.xml [hunk 2]
```diff
@@ -27,4 +27,6 @@
   <description>Fake Plugin To Run SonarLint UI tests with multiple flavors</description>
 
   <idea-version since-build="181.3007.14"/>
+
+  <depends>com.intellij.modules.lang</depends>
```
[the remark this answers is not in the document]
Comment (henryju) [reply its/src/main/resources/META-INF/plugin.xml:31]: There was an error in CLion logs, saying that plugins without dependencies are only supported in IDEA. Not sure it was a blocker, but this make logs cleaner.

File: its/src/test/kotlin/org/sonarlint/intellij/its/BaseUiTest.kt (hunk 1 above)
Comment (henryju) [reply its/src/test/kotlin/org/sonarlint/intellij/its/BaseUiTest.kt:146]: Indeed, but I think we can improve that later

**Pull request merged**